app/controllers/shortener/shortened_urls_controller.rb

```diff
@@ -18,7 +18,7 @@ def show
         ActiveRecord::Base.connection.close
       end
 
-      params.except! *[:id, :action, :controller]
+      params.to_h.except! *[:id, :action, :controller]
```

Comment:
I'm not sure this is going to work - doesn't this make a new hash that's a "copy" of params and mutate it?

Reply:
Good catch. You're right